packages/ui-components/package.json

```diff
@@ -37,17 +37,18 @@
     "@rollup/plugin-commonjs": "^24.0.0",
     "@rollup/plugin-node-resolve": "^15.1.3",
     "@rollup/plugin-typescript": "11.1.6",
-    "@storybook/addon-docs": "^7.6.20",
```

Comment:
It looks really good to me except for one thing:
The WiP/DateTimePicker story is broken because of an error with flatpickr (the headless date time picker package that we use internally). Error message is:
```
flatpickrInstanceRef.current.destroy is not a function
```

Reply:
I think I have idea what is wrong. I'll try to resolve it today

Reply:
I fixed it and added commit to this PR